packages/next/README.md

```diff
@@ -11,14 +11,21 @@
     <img alt="" src="https://img.shields.io/npm/l/next.svg?style=for-the-badge&labelColor=000000">
   </a>
   <a aria-label="Join the community on GitHub" href="https://github.com/zeit/next.js/discussions">
-    <img alt="" src="https://img.shields.io/badge/Join%20the%20community-blueviolet.svg?style=for-the-badge&logo=Next.js&labelColor=000000&logoWidth=20">
+    <img alt="" src="https://img.shields.io/badge/Join%20the%20community-blueviolet.svg?style=for-the-badge&logo=Next.js&labelColor=000000&logoWidth=20">  
+</p>
+<p align="center">
   </a>
+    <a aria-label="Join the spectrum chat" href="https://spectrum.chat/next-js">
+      <img alt="" src="https://withspectrum.github.io/badge/badge.svg">
+    </a>
 </p>
 
 ## Getting Started
 
 Visit <a aria-label="next.js learn" href="https://nextjs.org/learn">https://nextjs.org/learn</a> to get started with Next.js.
 
+Visit <a aria-label="next.js handbook" href="https://www.freecodecamp.org/news/the-next-js-handbook/">the Next.js Handbook on freeCodeCamp</a> to get familiar with how Next.js works.
```

Comment:
This is an unofficial community resource, it can be added to https://github.com/unicodeveloper/awesome-nextjs if it's not there yet.